### ISSUE TITLE
Update links in, "How to prepare for an accessibility staging review" page on Platform Website

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#35281]
## Issue Description
The[ How to prepare for an accessibility staging review](https://depo-platform-documentation.scrollhelp.site/developer-docs/How-to-prepare-for-an-accessibility-staging-review.1909293137.html#Howtoprepareforanaccessibilitystagingreview-Screenreadertests) documentation on Platform Website still links out to many github pages, when it should be linking to headings within the document.

For example, the **Read more about getting started with screen readers** link under [Advanced accessibility tests (recommended)](https://depo-platform-documentation.scrollhelp.site/developer-docs/How-to-prepare-for-an-accessibility-staging-review.1909293137.html#Howtoprepareforanaccessibilitystagingreview-Advancedaccessibilitytests(recommended)) should lead to [this section](https://depo-platform-documentation.scrollhelp.site/developer-docs/How-to-prepare-for-an-accessibility-staging-review.1909293137.html#Howtoprepareforanaccessibilitystagingreview-Screenreadertests) of the document, however instead it [links out to github.
](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#screen-reader-tests

---
## Tasks
- [ ] Test and assess all links on the page
- [ ] When a github link is found, determine if that content exists within the page itself
- [ ] If so, update the link to the correct location
- [ ] If not, keep the github link within the doc. However, document in this ticket what github links remain within the page.
   - we will need to transition these pages over to Platform Website if the information contained in the doc is  still valuable information for VFS teams

## Acceptance Criteria
- [ ] URL links on the How to prepare for an accessibility staging review page direct VFS teams to Platform Website guidance if the content exists within Platform Website

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by shiragoodman** [comment]
FYI @pjhill . I put both our team labels on this. This page is technically owned by QA Standards, but used by our team. Let me know if you have questions.

**Comment by CherylEvans** [comment]
@shiragoodman All links have been tested and redirected appropriately.  The following GH links are currently still in place and do not require a link redirection.

[Platform website footer feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=gmaciolek%2C+lianilychee&labels=content-ia-team%2C+pw-footer-feedback%2C+documentation-support&template=platform-website-footer-feedback.md)
[Cross-Platform Screen Reader Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/screenreader-testing-cross-platform.md)
[MacOS VoiceOver Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/mac-voiceover-guide.md)
[Screen reader best practices](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md#screen-readers)
[VoiceOver for macOS using Safari](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/mac-voiceover-guide.md)
[VA.gov supported browsers](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/request-for-comment/browser-support.md)
[Keyboard best practices](https://depo-platform-documentation.scrollhelp.site/developer-docs/How-to-prepare-for-an-accessibility-staging-review.1909293137.html)
[Learn more about automated and integrated 508 compliance](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/memos/508-ds-process.md#automated-and-integrated-508-compliance-tests-and-digital-service-reviews)
[508 Accessibility Issue Template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=508%2FAccessibility&template=508-issue.md&title=)
[Open accessibility issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/182)

**Comment by CherylEvans** [comment]
@shiragoodman 
I forgot to add the link with my changes: [Copy of How to prepare for an accessibility staging review](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2124283974/Copy+of+How+to+prepare+for+an+accessibility+staging+review)

**Comment by shiragoodman** [comment]
@CherylEvans Thanks. I made a couple minor comments in the draft doc. This document is actually in need of a huge overhaul... something we'll work on down the line. Thanks for updating the links!

**Comment by CherylEvans** [comment]
You are most welcome!  I would be happy to undertake the huge overhaul when
that time comes! :-)

Cheryl Evans | Consultant



The So Company <http://thesocompany.com/>

We make the government better.

Timezone: CST | GMT - 6

203-580-3529 | ***@***.***

SDVOSB | SBA 8(a) Certified | Digital Services Coalition Founding Member

**Comment by shiragoodman** [comment]
@chaseakirby @CherylEvans have the changes been published yet?

**Comment by CherylEvans** [comment]
@shiragoodman @chaseakirby Yes, they have.
